### DISCUSSION TITLE
Convert image to sRGB color profile

### DISCUSSION THREAD

**Discussion by leboeuf** [SixLabors/ImageSharp#1294] [Q&A] [answered]
How can I take an input image, for which I don't know the ICC profile in advance, and save it using an sRGB profile?

**Comment by JimBobSquarePants** [comment, marked as answer]
Assuming you mean jpeg, we always convert the colorspace to an rgb compatible colorspace which makes no assumption on the profile and is treated as sRGB.

However, and this is a big however. The conversion is done without ICC hints so Cmyk or YccKjpegs can often turn out incorrectly. We need someone to step up and complete this PR in order to apply accurate conversions.

https://github.com/SixLabors/ImageSharp/pull/273 

Once complete, each image format that supports ICC will have to have support added incrementally.
